packages/vite/src/node/logger.ts

```diff
@@ -158,19 +159,31 @@ export function createLogger(
   return logger
 }
 
-export function printServerUrls(
+export function printServerInfo(
```

Comment:
The name `printServerInfo` might no longer be what it does.
Do you come up with any good name?